clang/utils/TableGen/SveEmitter.cpp

```diff
@@ -383,6 +381,20 @@ class SVEEmitter {
                        SmallVectorImpl<std::unique_ptr<Intrinsic>> &Out);
 };
 
+const std::array<SVEEmitter::ReinterpretTypeInfo, 12> SVEEmitter::Reinterprets =
+    {{{SVEType("c"), "s8"},
```

Comment:
Can we change this to be SVEType("c", 'd'), so we don't need to change in the class in line 76.
We can still keep SVEType(StringRef TS, char CharMod, unsigned NumVectors = 1).
My line of thought is to reduce the changes in this patch

Reply:
Ok, I'll do that.

Reply:
Done